src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Stream.cs

```diff
@@ -21,6 +21,9 @@ public static partial class JsonSerializer
         /// <param name="cancellationToken">The <see cref="System.Threading.CancellationToken"/> which may be used to cancel the write operation.</param>
         public static Task SerializeAsync<TValue>(Stream utf8Json, TValue value, JsonSerializerOptions? options = null, CancellationToken cancellationToken = default)
         {
+            if (utf8Json == null)
```

Comment:
Nice catch. We should update the docs with the new exceptions:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.serializeasync?view=netcore-3.1#System_Text_Json_JsonSerializer_SerializeAsync_System_IO_Stream_System_Object_System_Type_System_Text_Json_JsonSerializerOptions_System_Threading_CancellationToken_

If you get the chance, please add the exceptions this (and other SerializeAsync overloads) throw, here:
https://github.com/dotnet/dotnet-api-docs/blob/1e6b2e6052185c6744054acc10cd06e435a4fb8b/xml/System.Text.Json/JsonSerializer.xml#L546-L563

See other overloads that have exception docs as an example:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.serialize?view=netcore-3.1#System_Text_Json_JsonSerializer_Serialize__1_System_Text_Json_Utf8JsonWriter___0_System_Text_Json_JsonSerializerOptions_

cc @carlossanlop

Reply:
Will do, thanks for the pointers to the docs code and example.

Reply:
Created issue https://github.com/dotnet/dotnet-api-docs/issues/3961